src/confd/src/ietf-routing.c

```diff
@@ -0,0 +1,149 @@
+
```

Comment:
Every new C-file, including headers, should have a line declaring/reasserting the license:

    /* SPDX-License-Identifier: BSD-3-Clause */